crates/re_space_view_spatial/src/visualizers/transform3d_arrows.rs

```diff
@@ -114,20 +121,21 @@ const AXIS_COLOR_Y: Color32 = Color32::from_rgb(0, 240, 0);
 const AXIS_COLOR_Z: Color32 = Color32::from_rgb(80, 80, 255);
 
 pub fn add_axis_arrows(
-    line_builder: &mut LineStripSeriesBuilder,
+    line_builder: &mut re_renderer::LineDrawableBuilderAllocator<'_>,
     world_from_obj: macaw::Affine3A,
     ent_path: Option<&EntityPath>,
     axis_length: f32,
     outline_mask_ids: re_renderer::OutlineMaskPreference,
-) {
+) -> Result<(), re_renderer::renderer::LineDrawDataError> {
     use re_renderer::renderer::LineStripFlags;
 
     // TODO(andreas): It would be nice if could display the ViewCoordinates axis names (left/right/up) as a tooltip on hover.
 
     let line_radius = re_renderer::Size::new_points(1.0);
 
     let mut line_batch = line_builder
-        .batch("axis_arrows")
+        .reserve(3, 6)?
```

Comment:
maybe we can name this more descriptive?

```suggestion
        .reserve_batches_and_vertices(3, 6)?
```

Reply:
in fact does not reserve batches, but yeah `strips_and_vertices` would work